### PR TITLE
Rename reverb attributes to use "reverb_" prefix instead of "rev_" [Fix #115]

### LINKED ISSUE #115: rev_sync attribute should be named sync_to_reverb to be intuitive
Currently the "Sync to Reverb" attribute code is `rev_sync`, which is somewhat unclear.  I'd suggest using `sync_to_reverb` to align with the UI name and to at least make it explicit that it's a Reverb attribute.

### PR DESCRIPTION
**Pull request by StevenWolfe**
Tested migration script against 0.4.4

### CODE REVIEW

**Comment by StevenWolfe** [comment]
closing temporarily - looks like the squash may have an error

**Comment by StevenWolfe** [comment]
False alarm. Not enough coffee.

File: app/code/community/Reverb/ReverbSync/sql/reverbsync_setup/mysql4-upgrade-0.1.6-0.1.7.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
```
Comment (skwp) [inline app/code/community/Reverb/ReverbSync/sql/reverbsync_setup/mysql4-upgrade-0.1.6-0.1.7.php]: i think the version number in this file is now out of date with other migrations. can I just change the file name safely?

**Comment by StevenWolfe** [comment]
The `<version>` value in `config.xml` will need to be updated, and the file renamed.  It would also be good to do a code-wide search for any usages of the old attribute names after the merge:

 - RevProductId -> ReverbProductId
 - rev_product_id -> reverb_product_id
 - RevProductUrl -> ReverbProductUrl
 - rev_product_url -> reverb_product_url
 - RevSync -> ReverbSyncEnabled
 - rev_sync -> reverb_sync_enabled

**Comment by skwp** [comment]
Thanks Steve. I think this PR is a nice to have rename that doesn't really impact anything but does put a lot of burden on us not to break any existing customers. I'm inclined not to merge it given that I can't guarantee its safety..

**Comment by StevenWolfe** [comment]
no problem, we're using it but I can convert back to the old attribute names--or I can double-check the safety and redo the pull request in a while

**Comment by skwp** [comment]
if you redo the PR that would be a big help. again I appreciate your work, I just don't want to be put in the position of breaking other customers by accident

**Comment by dunagan5887** [comment]
I would agree that this is a risky merge due to the existing client instances

**Comment by skwp** [comment]
sorry, decided to close this for the time being due to the risk/reward quotient :)

**Pull request closed without merging**